src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -68,6 +69,8 @@ export const PublishingAccordion = ({
   const [isLoading, setIsLoading] = useState(LoadingState.None);
   const [registrationIsValid, setRegistrationIsValid] = useState(false);
 
+  const registrationHasFile = registration.associatedArtifacts.some((artifact) => associatedArtifactIsFile(artifact));
```

Comment:
er det ikke egentlig denne du er interessert i? om brukeren nå legger til en ny fil etter å ha fått godkjent en ticket, vil det stå at filer er publisert, selv om det ikke er det (har ikke sjekka i praksis, men frmestår sånn)
```suggestion
  const registrationHasFile = registration.associatedArtifacts.some((artifact) => artifact.type === "PublishedFile");
```